chain/neutrino_test.go

```diff
@@ -0,0 +1,209 @@
+package chain
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestNeutrinoClientSequentialStartStop ensures that the client
+// can sequentially Start and Stop without errors or races.
+func TestNeutrinoClientSequentialStartStop(t *testing.T) {
+	var (
+		timeout      = time.After(1 * time.Second)
```

Comment:
CI can be very laggy, almost as if the VMs randomly pause, but if 1sec works, ok.  I'd be surprised if that's not often too short.  Would it hurt to go with 5 or even 10 seconds?  It only slows a test that would fail, not a passing one.